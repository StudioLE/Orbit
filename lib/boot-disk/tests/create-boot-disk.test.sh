#!/bin/bash
set -uo pipefail

# ARGS
ACTUAL_FILE=$1

# SETUP

source /srv/lib/boot-disk/create-boot-disk.sh

# START

create-boot-disk "ubuntu" "jammy" "20G"
echo "${OUTPUT}" > "${ACTUAL_FILE}"
