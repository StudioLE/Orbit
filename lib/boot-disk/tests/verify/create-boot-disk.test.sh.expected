/srv/artifacts/ubuntu/jammy/jammy-server-cloudimg-amd64-20G.qcow2
